# This service spawns the described model in the unreal world.


# Description of the model to be spawned.
# The class_name of the model_descriptions instance_id
# and the model_descriptions pose need to be set.
# Everything else is optional.
# If instance_id.id is not set it will be generated and set in 
# returned instance_id
unreal_msgs/ModelDescription model_description
---
# The returned instance_id will contain the generated id,
# if the id was set beforehand this will be equal to the
# instance_id in the model_description of the request.
unreal_msgs/InstanceId instance_id
# true if model was spawned successfully.
bool success
